### PR TITLE
Adding FunctionsNetHost to worker repo

### PR DESCRIPTION
**Pull request by kshyju**

### CODE REVIEW

**Review by brettsam — APPROVED** [review]

File: host/src/Protos/identity/ClaimsIdentityRpc.proto [hunk 1]
```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
```
[the remark this answers is not in the document]
Comment (fabiocav) [reply host/src/Protos/identity/ClaimsIdentityRpc.proto:1]: Agreed. Can we have a follow up PR with that change?

**Review by fabiocav — APPROVED** [review]
Small nit comments

File: host/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# net-worker-host
+
+[![Build Status](https://azfunc.visualstudio.com/Azure%20Functions/_apis/build/status/fabiocav.net-worker-host?branchName=main)](https://azfunc.visualstudio.com/Azure%20Functions/_build/latest?definitionId=201&branchName=main)
```
[the remark this answers is not in the document]
Comment (kshyju) [reply host/README.md:3]: will do after the new pipeline is setup.

File: host/src/Protos/identity/ClaimsIdentityRpc.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (kshyju) [reply host/src/Protos/identity/ClaimsIdentityRpc.proto:1]: Opened #1121 for the proto file unification.

**Pull request merged**